src/main/java/com/feedzai/commons/sql/abstraction/engine/DatabaseEngine.java

```diff
@@ -132,6 +132,16 @@
      */
     void dropEntity(final DbEntity entity) throws DatabaseEngineException;
 
+    /**
+     * Drops a view.
+     *
+     * @param view The view name.
+     * @throws DatabaseEngineException If something goes wrong while dropping the view.
+     *
+     * @since 2.5.3
+     */
+    void dropView(final String view) throws DatabaseEngineException;
```

Comment:
Hello @nuno-santos96 (cc @dfialho and @joaonuno),
This is a breaking change, it will break the code when extending DatabaseEngine class.